### PR TITLE
Implement getRelativeDateString

### PR DESCRIPTION
**Pull request by ryotogashi**

### CODE REVIEW

**Comment by ryotogashi** [comment]
@axross Can you check when you have time?

File: utilities/getRelativeDateString.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+export default function getRelativeDateString(date: Date): string {
```
Comment (axross) [inline utilities/getRelativeDateString.ts]:
```suggestion
export function getRelativeDateString(date: Date): string {
```

Comment (axross) [reply utilities/getRelativeDateString.ts]: Or you can make `getRelativeDateString` and `getRelativeShortDateString` in separated files

**Comment by ryotogashi** [comment]
@axross Thank you for your review! I fixed it!

**Comment by axross** [comment]
Seems the function to calculate final pot size is included. Is this intentional? 🤔

**Comment by ryotogashi** [comment]
@axross  Sorry what do you mean? I merged other functions but not calculateFinalpot function.

**Comment by axross** [comment]
@ryotogashi Can you look at changes?

**Pull request merged**